index.html

```diff
@@ -39,16 +39,16 @@ <h2 class="h2-white">
 			</div>
 			<ul class="list-icons animate">
 				<li>
-					<a href="#"><img src="Images/GithubIcon.svg" class="trans"></a>
+					<a href="#"><img src="Images/GithubIcon.svg" alt="Github logo"></a>
 				</li>
 				<li>
-					<a href="#"><img src="Images/LinkedInIcon.svg"></a>
+					<a href="#"><img src="Images/LinkedInIcon.svg" alt="Linkedin Logo"></a>
```

Comment:
- [ ] Please remove the text "logo", even don't use the words like `button`, `icon`, `image of` and `logo` in your **alternative-text** because by default when  a person who use screen-readers, screen reader automatically provide the context. Kindly use "LinkedIn Profile" because it will assist the person to know whether he/she is interested to look into your `profile` or not.